class_person.rb

```diff
@@ -1,14 +1,19 @@
-class Person
+require_relative 'class_nameable'
```

Comment:
- [ ] Good job for importing the nameable class, however, your file name reads `nameable` not `class_nameable` 🐛 Kinfly make sure to fix this. Currently I'm getting the following error 👇

![image](https://github.com/BenMKT/OOP-school-library/assets/106119105/cd0e5943-3d9d-40aa-83af-98bad95719f1)

Reply:
- Refactor Person class to resolve bug